docs/code/cloud_storage.py

```diff
@@ -13,7 +13,8 @@
 from tensorbay import GAS
 
 gas = GAS("Accesskey-*****")
-cloud_client = gas.get_cloud_client("config_name")
+images_cloud_client = gas.get_cloud_client("images_config_name")
+labels_cloud_client = gas.get_cloud_client("labels_config_name")
```

Comment:
why we need two configs for images and labels

Reply:
Because the image file and label file are at different path, so we need two configs.